source/parsing/Preprocessor.cpp

```diff
@@ -1091,6 +1096,18 @@ Trivia Preprocessor::handleNoUnconnectedDriveDirective(Token directive) {
     return createSimpleDirective(directive);
 }
 
+Trivia Preprocessor::handleCellDefineDirective(Token directive) {
+    checkOutsideDesignElement(directive);
```

Comment:
The LRM says "These directives may appear anywhere in the source description, but it is recommended that the directives be specified outside any design elements." so we shouldn't actually enforce that they appear outside a design element.

Reply:
Thanks, I missed that.